Pipfile

```diff
@@ -29,6 +29,7 @@ allow_prereleases = true
 [scripts]
 start = "python opensir-cli"
 lint = "pylint **/*.py"
+markdownlint = "npx markdownlint-cli README.md"
```

Comment:
So you can also check it in your local machines. Be advised that this requires node to be installed in your machines to run. Most modern linux distros and OSX come with this installed as far as I understand.

If you do not have node installed, it's no problem. Worst case scenario you won't be able to run the command locally, but the CI will still do it.

Reply:
Btw, check https://github.com/open-sir/open-sir/issues/72